### PR TITLE
Handle wrapped grpc error in funcutil.IsGrpcError

### LINKED ISSUE #17546: [Bug]: func.IsGrpcError cannot handle wrapped grpc error
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Environment

```markdown
- Milvus version:40326f0
- Deployment mode(standalone or cluster):
- SDK version(e.g. pymilvus v2.0.0rc2):
- OS(Ubuntu or CentOS): 
- CPU/Memory: 
- GPU: 
- Others:
```

### Current Behavior

If grpc error is wrapped by `%w` verb, `funcutil.IsGrpcError` cannot judge error type correctly.

### Expected Behavior

`funcutil.IsGrpcError` shall return true for wrapped grpc error

### Steps To Reproduce

_No response_

### Milvus Log

_No response_

### Anything else?

Related to #17367

### PR DESCRIPTION
**Pull request by congqixia**
Fix #17546 

/kind improvement

Signed-off-by: Congqi Xia <congqi.xia@zilliz.com>

### CODE REVIEW

**Comment by jiaoew1991** [comment]
LGTM

File: internal/util/funcutil/func.go [hunk 1]
```diff
@@ -360,6 +360,14 @@ func IsGrpcErr(err error) bool {
 	if err == nil {
 		return false
 	}
-	_, ok := grpcStatus.FromError(err)
-	return ok
+	for {
+		_, ok := grpcStatus.FromError(err)
```
Comment (yah01) [inline internal/util/funcutil/func.go:364]: It may be better to move the nil check at 369 line here, and remove the nil check at 360 line.

Comment (congqixia) [reply internal/util/funcutil/func.go:364]: Fixed

**Pull request merged**